modules/openthread/Kconfig.features

```
@@ -235,6 +241,9 @@ config OPENTHREAD_SRP_SERVER
 	select OPENTHREAD_NETDATA_PUBLISHER
 	select OPENTHREAD_ECDSA
 
+config OPENTHREAD_TCP
```

Please align it with `OPENTHREAD_TCP_ENABLE` and `zephyr/modules/openthread/platform/openthread-core-zephyr-config.h`

---

```suggestion
config OPENTHREAD_TCP
```
please align with `OPENTHREAD_EXCLUDE_TCPLP_LIB`